site/src/Librecores/ProjectRepoBundle/DataFixtures/ORM/LoadDevTestData.php

```diff
@@ -118,4 +125,47 @@ private function createUser($username, $password)
 
         return $user;
     }
+
+    /**
+     * Populate an ClassificationHierarchy object
+     *
+     * @param integer $parent_id
+     * @param string $name
+     * @return \Librecores\ProjectRepoBundle\Entity\ClassificationHierarchy
+     */
+    private function createClassifier($parent_id,$name) {
```

Comment:
we use camelCase for variables. Add a space between the variables, and place the opening { in a new line.